web-components/src/components/folksnomy/folksonomy-properties.ts

```diff
@@ -0,0 +1,785 @@
+            />
+          </td>
+          <td></td>
+          <td>
+            ${this.renderDualRangeSlider(
+              "count",
+              this.ranges.countMin,
+              this.ranges.countMax,
+              this.filters.countMin,
+              this.filters.countMax
+            )}
+          </td>
+          <td>
+            ${this.renderDualRangeSlider(
+              "values",
+              this.ranges.valuesMin,
+              this.ranges.valuesMax,
+              this.filters.valuesMin,
+              this.filters.valuesMax
+            )}
+          </td>
+        </tr>
+      </thead>
+    `
+  }
+
+  private renderPropertyRow(property: { k: string; count: number; values: number }) {
+    return html`
+      <tr class="property">
+        <td></td>
+        <td>
+          <a href="${this.getPropertyUrl(property.k)}" class="property-name"> ${property.k} </a>
+        </td>
+        <td>
+          <a
+            href="${this.getDocumentationUrl(property.k)}"
+            class="doc-link"
+            title="Documentation for ${property.k}"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            🛈
+          </a>
+        </td>
+        <td class="count">${property.count}</td>
+        <td class="values">${property.values}</td>
+      </tr>
+    `
+  }
+
+  private renderContent() {
+    if (this.loading) {
+      return html`<div class="loading">Loading properties...</div>`
+    }
+
+    if (this.error) {
+      return html`<div class="error">${this.error}</div>`
+    }
+
+    if (this.properties.length === 0) {
+      return html` <div class="empty-state">No properties found.</div> `
+    }
+
+    return html`
+      <div class="filter-controls">
+        <div class="rows-counter">
+          Rows: ${this.filteredProperties.length} / ${this.properties.length}
+        </div>
+        <div class="button-group">
+          <button class="download-btn" @click="${this.downloadCSV}">Download CSV</button>
+          <button class="reset-btn" @click="${this.resetFilters}">Reset</button>
+        </div>
+      </div>
+      <table class="properties-table" id="folksonomy-properties-table">
+        ${this.renderTableHeader()}
+        <tbody id="free_prop_body">
+          ${this.filteredProperties.map((property) => this.renderPropertyRow(property))}
+        </tbody>
+      </table>
+    `
+  }
+
+  override render() {
+    return html`
+      <div class="properties-container">
+        <h2 id="property_title">Properties</h2>
+        <p>
+          Open Food Facts allows anyone to reuse contributed properties or create new ones (see the
+          <a
+            href="https://wiki.openfoodfacts.org/Folksonomy_Engine"
+            target="_blank"
+            rel="noopener noreferrer"
+            >Folksonomy Engine project</a
+          >). Here's the list of all contributed properties.
+        </p>
```

Comment:
It should be a slot, it will be easier to handle translation in openfoodfacts-server this way, and it would also be more natural (this explanation may depend on the context).